_includes/install_instructions/shell.html

```diff
@@ -26,8 +26,11 @@ <h3>The Bash Shell</h3>
               </li>
               <li>
                 <strong>
-                  From the dropdown menu select "Use the Nano editor by default" (NOTE: you will need to scroll <emph>up</emph> to find it) and click on "Next".
+                  From the dropdown menu select "Use the nano editor by default" (NOTE: you may need to scroll <em>up</em> to find it) and click on "Next".
                 </strong>
+              </li>
+		<li>
+                  Ensure that "Let Git decide" is selected and click on "Next".
```

Comment:
This part is not needed. It's present at https://github.com/carpentries/workshop-template/pull/712/files#diff-3e9bddf2885218ffef82ffe85c1a15cb40879675df56feeb19e1bb043f8bc4caR37-R38

Reply:
It was added in #711